### ISSUE TITLE
ERROR: Invalid value for argument cmd and update_gtags.sh is not executable

### ISSUE THREAD

**Issue by rockyzhang24** [ludovicchabant/vim-gutentags#322]
**Describe the bug**

See the screenshot below please

<img width="1725" alt="image" src="https://user-images.githubusercontent.com/11582667/168412695-7aa0fa45-3a0b-41cc-b7a1-a5310afa8ee8.png">

**Steps to reproduce**
Open any file ( I opened a go file)

**Share your setup**
- What OS and version of Vim are you using? 
	macOS 12.3.1
	neovim version: NVIM v0.8.0-dev+1660-g9e4286294
- What version of `ctags`, `gtags`, or whatever do you have installed?
	Universal Ctags 5.9.0
	gtags (Global) 6.6.8
- Are you using `g:gutentags_cache_dir`?
	Yes. I set it to `~/.cache/tags/`. However, if I don't set it, this issue still exists.

**Post the logs**
<img width="1725" alt="image" src="https://user-images.githubusercontent.com/11582667/168413049-8fb17416-c269-44f2-b597-f6e2f4f2a823.png">

**Additional context**
Nothing

Thank you very much.

**Comment by ragu-manjegowda** [comment]
I was able to fix this with the following workaround,

```
chmod u+x PATH_TO_PLUGINS_DIR/vim-gutentags/plat/unix/update_gtags.sh
```

Tested on Arch and Ubuntu

**Comment by MrZLeo** [comment]
> I was able to fix this with the following workaround,
> 
> ```
> chmod u+x PATH_TO_PLUGINS_DIR/vim-gutentags/plat/unix/update_gtags.sh
> ```
> 
> Tested on Arch and Ubuntu

Tested on macOS

**Comment by wow-yes** [comment]
> ```
> chmod u+x PATH_TO_PLUGINS_DIR/vim-gutentags/plat/unix/update_gtags.sh
> ```

In fact, all the `.sh` files in this dir should be executable.

```
chmod u+x PATH_TO_PLUGINS_DIR/vim-gutentags/plat/unix/*.sh
```

**Comment by ludovicchabant** [comment]
This is fixed with 865d17a63b70690beee855e72abd38db278c1839. Sorry for the delays.
